playbooks/ha/post_install.yml

```diff
@@ -47,9 +47,11 @@
         --images='{{satellite_hostname}}:5000/{{org_label}}-fusor-ose-haproxy-router' \
         --selector='region=infra' \
         --replicas='2'"
+      ignore_errors: yes
```

Comment:
Might want to consider a "failed_when" here in the long term, let's you parse the output of the command for the specific thing you want to ignore, rather than ignoring all errors. Could also add tasks that query to see if this has been performed already and skip if true.